pkg/odo/cli/component/create.go

```diff
@@ -954,10 +955,18 @@ func (co *CreateOptions) downloadProject(projectPassed string) error {
 		return err
 	}
 
+	if co.devfileMetadata.starterToken == "" && registryUtil.IsSecure(co.devfileMetadata.devfileRegistry.Name) {
+		token, err := keyring.Get(util.CredentialPrefix+co.devfileMetadata.devfileRegistry.Name, "default")
```

Comment:
use `fmt.Sprintf` here. Should `default` be a const?

Reply:
Done